lib/foundation_rails_helper/form_builder.rb

```diff
@@ -129,6 +129,42 @@ def custom_label(attribute, text, options, &block)
       label(attribute, text, options)
     end
 
+    def tag_from_options(name, options)
+      if options && options[:size].present? && options[:value].present?
```

Comment:
A nitpick, use guard clauses instead of if-else when you can.
```ruby
return ''.html_safe unless options &&
                           options[:size].present? &&
                           options[:value].present?

content_tag(:div, ...
```